recipe/meta.yaml

```diff
@@ -22,6 +19,18 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - cmake             {{ cmake       }}
+    - dataclasses       0.7               #[py == 36]
```

Comment:
Most of these should be under `host` instead of build. I think `cmake` and the C compilers are the only things that should be under `build`.

Reply:
They were only under host originally. But they still failed. I had to add them to build.